lib/benchee/conversion/memory.ex

```diff
@@ -0,0 +1,197 @@
+defmodule Benchee.Conversion.Memory do
+  @moduledoc """
+  Unit scaling for memory converting from bytes to kilobytes and others.
+  """
+
+  alias Benchee.Conversion.{Format, Scale, Unit}
+
+  @behaviour Scale
+  @behaviour Format
+
+  @bytes_per_kilobyte 1024
+  @bytes_per_megabyte @bytes_per_kilobyte * @bytes_per_kilobyte
+  @bytes_per_gigabyte @bytes_per_megabyte * @bytes_per_kilobyte
+  @bytes_per_terabyte @bytes_per_gigabyte * @bytes_per_kilobyte
+
+  @units %{
+      terabyte:        %Unit{
```

Comment:
formatting is a bit curious here, e.g. no other unit seems to be so big to need to indent that far.